api/internal/managers/linux/installation/config.go

```diff
@@ -233,3 +236,62 @@ func (m *installationManager) ConfigureHost(ctx context.Context, rc runtimeconfi
 
 	return nil
 }
+
+// CalculateRegistrySettings calculates registry settings for airgap installations
+func (m *installationManager) CalculateRegistrySettings(ctx context.Context, releaseData *release.ReleaseData) (*types.RegistrySettings, error) {
```

Comment:
release data should be passed as an option to the manager similar to all other managers since it's not mutable